packages/test-globals/src/index.ts

```diff
@@ -63,5 +68,25 @@ declare global {
      * Wraps an object type with Jest mock type definitions.
      */
     export type MockedObject<T extends object> = JestMockedObject<T>;
+    /**
+     * Constructs the type of a spied class or function.
+     */
+    export type Spied<T extends ClassLike | FunctionLike> = JestSpied<T>;
```

Comment:
this is copied manually, right? Should we set up a script which keeps it in sync with `@jest/globals`?

Reply:
Would be possible, but is it worth it? I mean, these types are only used in Jest repo. If something will be missing, typechecks of tests will catch it for us.

Perhaps some check that these types are still exposed in `@jest/globals`? Just another type test? Sound simple to do and to run automatically. Together with typechecks of test files that should be enough. I will do this as separate PR. If that’s fine? (;

Reply:
Definitely separate 👍